index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <main class="main-container">
+      <header class="toolbar">
+        <a>Welcome</a>
+        <span> <img src="images/toggle.png" alt="toggle icon" /> </span>
```

Comment:
- [ ] Please fix the position of the logo 
![image](https://user-images.githubusercontent.com/74744787/159047226-e87b3ecf-a446-49fa-8728-1ca14ed73eb3.png)

Reply:
I dont understand this, can you elaborate